firestore/app/src/main/java/com/google/firebase/example/fireeats/java/RestaurantDetailFragment.java

```diff
@@ -12,6 +12,7 @@
 import androidx.annotation.Nullable;
 import androidx.appcompat.app.AppCompatActivity;
 import androidx.fragment.app.Fragment;
+import androidx.navigation.fragment.findNavController;
```

Comment:
The correct java import is `NavController`:
```suggestion
import androidx.navigation.NavController;
```

Reply:
Sure, got distracted. Thanks.